gradle.properties

```diff
@@ -6,5 +6,5 @@ spinnakerGradleVersion=8.10.0
 pf4jVersion=3.4.1
 korkVersion=7.85.0
 orcaVersion=8.12.0
-kotlinVersion=1.4.10
+kotlinVersion=1.3.72
```

Comment:
Why does this need to change?

Reply:
I updated the Kotlin version in a [previous update](https://github.com/pulumi/spinnaker-preconfigured-job-plugin/commit/3a9cb1ae5e67da444b3fc6a96bb40a43460048aa#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19) to `master` to fix a cron schedule failure. But it turned out to be a large issue than just updating the version, which is what this PR fixes. So I undid the Kotlin version change here.